Realm.Shared/Realm.cs

```diff
@@ -420,10 +333,10 @@ static public void DeleteRealm(RealmConfiguration configuration)
 
 
         [ReliabilityContract(Consistency.WillNotCorruptState, Cer.Success)]
-        private TableHandle GetTable(Type realmType)
+        private TableHandle GetTable(Schema.Object schema)
         {
             var result = new TableHandle();
-            var tableName = "class_" + realmType.Name;
+            var tableName = "class_" + schema.Name;
```

Comment:
This should eventually use `ObjectStore::table_name_for_object_type` rather than constructing table names itself.